### PR TITLE
[service.subtitles.torec] 1.1.0

### PR DESCRIPTION
**Pull request by slmosl**
### Description
updated the addon to support new parent website version (7.0)

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a sinlge commit with using the following style: [sciprt.foo.bar] v1.0.0

Addtional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuesly monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: service.subtitles.torec/addon.xml [hunk 1]
```diff
@@ -1,8 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="service.subtitles.torec"
        name="Torec"
-       version="1.0.10"
-       provider-name="slmosl">
```
Comment (phil65) [inline service.subtitles.torec/addon.xml]: provider-name should stay.

**Comment by phil65** [comment]
One comment, rest looks good. Thx!

**Comment by phil65** [comment]
Please squash commits.

**Comment by slmosl** [comment]
@phil65 done

**Comment by phil65** [comment]
Thx!

**Pull request merged**